docker-compose.yaml

```diff
@@ -0,0 +1,60 @@
+version: '3.7'
+
+services:
+
+  db:
+    image: mariadb:10.4
+    restart: on-failure
+    environment:
+      MYSQL_DATABASE: testdb
+      MYSQL_USER: wordpress
+      MYSQL_PASSWORD: wordpress
+      MYSQL_RANDOM_ROOT_PASSWORD: 'yes'
+    volumes:
+      - db:/var/lib/mysql
+
+  wordpress-woocommerce-blocks-dev:
+    depends_on:
+      - db
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - 8084:80
+    restart: on-failure
+    environment:
+      WORDPRESS_DB_HOST: db
+      WORDPRESS_DB_NAME: testdb
+      WORDPRESS_DB_USER: wordpress
+      WORDPRESS_DB_PASSWORD: wordpress
+      WORDPRESS_TABLE_PREFIX: "wp_"
+      WORDPRESS_DEBUG: 1
+    volumes:
+      - "./:/var/www/html/wp-content/plugins/woocommerce-gutenberg-products-block"
+      - "./vendor/woocommerce/woocommerce/:/var/www/html/wp-content/plugins/woocommerce"
+      - wordpress:/var/www/html
+
+  wordpress-cli:
+    depends_on:
```

Comment:
Are we using wp-cli or is this an experiment that didn't work?

Reply:
yes we're using it to activate plugins